modules/Microsoft.DBforPostgreSQL/flexibleServers/.test/private/deploy.test.bicep

```diff
@@ -0,0 +1,99 @@
+targetScope = 'subscription'
```

Comment:
Probably also in this case we could merge private and public tests into a more general common test, validating the majority of parameter inputs. Only an open question if this should be done in this PR or as a separated issue.

Reply:
Agree, let's move the discussion to a separate issue #2232